AccessCheckoutSDK/AccessCheckoutSDKPactTests/AccessCheckoutSDKPactTests.swift

```diff
@@ -13,13 +13,20 @@ class AccessCheckoutSDKPactTests: XCTestCase {
                                                 consumer: "access-checkout-iOS-sdk")
 
     class MockDiscovery: Discovery {
+        var serviceEndpoint: URL?
+        
         var verifiedTokensSessionEndpoint: URL?
```

Comment:
Question for you - Do we need `verifiedTokensSessionEndpoint` if we have `serviceEndpoint`?